base_layer/wallet_ffi/src/lib.rs

```diff
@@ -3313,6 +3316,7 @@ pub unsafe extern "C" fn wallet_create(
     callback_store_and_forward_send_result: unsafe extern "C" fn(c_ulonglong, bool),
     callback_transaction_cancellation: unsafe extern "C" fn(*mut TariCompletedTransaction, u64),
     callback_txo_validation_complete: unsafe extern "C" fn(u64, bool),
+    callback_contacts_liveness_data_updated: unsafe extern "C" fn(*mut TariContactsLivenessData),
```

Comment:
You have not provided the means for an FFI client to destructure the contents of the TariContactsLivenessData into FFI primitives.

Reply:
Thanks, well spotted. That was a huge omission.